lagtraj/input_definitions/examples_files/forcings_conversion/dephy.yaml

```diff
@@ -21,23 +21,8 @@ rad_thetal      : 0
 forc_omega      : 0
 forc_w          : 1
 forc_geo        : 1
-nudging_u       : 0
-nudging_v       : 0
-nudging_temp    : 0
-nudging_theta   : 0
-nudging_thetal  : 0
-nudging_qv      : 0
-nudging_qt      : 0
-nudging_rv      : 0
-nudging_rt      : 0
 surfaceType     : ocean
 surfaceForcing  : ts
 surfaceForcingWind               : z0_traj
-#nudging_method_scalar_traj       : cos
-#nudging_time_scalar_traj         : 10800.
-#nudging_height_scalar_traj       : 600.
-#nudging_transition_scalar_traj   : 200.
-#nudging_method_momentum_traj     : cos
-#nudging_time_momentum_traj       : 10800.
-#nudging_height_momentum_traj     : 600.
-#nudging_transition_momentum_traj : 200.
+nudging_method_scalars           : false
```

Comment:
wouldn't it be clearer to have the default value be `None` to indicate that no nudging method should be used? `False` isn't a function, although I understand what you mean the name of the parameter suggest to me that `nudging_method_scalars` is referring to method used, not whether or not a method should be used

Reply:
I had issues with none values here before. Would it be OK to use this approach for now?